conda/recipes/libcudf/meta.yaml

```diff
@@ -33,7 +33,9 @@ build:
 
 requirements:
   build:
-    - cmake >=3.17.0
+    - cmake >=3.18
+    - gtest
+    - gmock
```

Comment:
These should be in `host` instead of `build`. The `build` section is specifically for build tools like cmake, compilers, etc. that you need for your native platform (if you were cross compiling).